lib/google_scraping/dashboard/keywords.ex

```diff
@@ -42,15 +43,73 @@ defmodule GoogleScraping.Dashboard.Keywords do
   Creates keywords list.
   """
   def create_keyword_list(keyword_list, user) do
-    Enum.each(keyword_list, fn keyword_name ->
-      keyword_params = %{
-        user_id: user.id,
-        name: keyword_name,
-        status: :new
-      }
-
-      # TODO: start background jobs for each created keyword
-      create_keyword(keyword_params)
+    keywords_data =
```

Comment:
How about moving these keywords_data building in a separate private function?
```suggestion
    keywords_data = build_keywords_data(..)
```